FirebasePerformance.podspec

```diff
@@ -55,9 +54,9 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
     'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"',
   }
 
-  s.ios.framework = 'CoreTelephony'
-  s.ios.framework = 'QuartzCore'
-  s.ios.framework = 'SystemConfiguration'
+  s.framework = 'CoreTelephony'
```

Comment:
Does this framework exist for tvOS? If not, we might have to create a platform based framework inclusion.

Reply:
Good catch! Thanks.